CSS/style.css

```diff
@@ -41,7 +43,6 @@ body {
   min-height: 100vh;
   height: max-content;
   margin: 0 auto;
-  font-size: 62.5%;
 }
 
 h1 {
```

Comment:
try & use class or id instead of element, because when someone would use your component library their elements would get the styles that they may not want.